config/config.fs2.hocon.sample

```diff
@@ -0,0 +1,23 @@
+auth = {
+  type = "Gcp"
```

Comment:
- Acronyms are generally all-caps. Is there a specific reason to use `Gcp`?
- From a user's perspective, it'd be useful if we could see valid values of configuration fields, e.g. is `gcp` or `GCP` valid here? Or do we want to rely on user-friendly error messages explaining what's wrong and how to fix?

Reply:
- It'd be nice to see which fields are optional and and which values are used by default when applicable e.g. `assetsUpdatePeriod` if it is not configured

Reply:
Spelling was dictated by codecs deriivation (it uses a exact case class name). I decided no to change it for now as `Gcp` is the only valid value here, but I agree this is something that should be fixed.

I'll add comments to the config file.